components/scream/cime_config/testdefs/testmods_dirs/scream/small_kernels/shell_commands

```diff
@@ -0,0 +1 @@
+./xmlchange SCREAM_CMAKE_OPTIONS='SCREAM_SMALL_KERNELS On'
```

Comment:
Does this line definitely work? I don't remember whether you then need to then reset tracer count to 10.

Reply:
@ambrad , wow, good catch. I have fixed this and updated the documentation for making SCREAM_CMAKE_OPTIONS test mods.

Reply:
Yeah, I think we need `--append` here.